### ISSUE TITLE
Error in crc start

### ISSUE THREAD

**Issue by christianorios** [code-ready/crc#2389]
Please, would anyone help me? Ive got this after ./crc start in Windows 10 Pro:
```
PS C:\temp> ./crc start --log-level debug
DEBU CodeReady Containers version: 1.27.0+3d6bc39d
DEBU OpenShift version: 4.7.11 (embedded in executable)
DEBU Running 'crc start'
DEBU Total memory of system is 17178800128 bytes
DEBU No new version available. The latest version is 1.27.0
Is 'crc daemon' running? Cannot reach daemon API: Get "http://unix/api/version": open \\.\pipe\crc-http: O sistema não pode encontrar o arquivo especificado.
PS C:\temp>
```

**Comment by praveenkumar** [comment]
@christianorios Have you perform the `crc setup` before `crc start` ?

**Comment by christianorios** [comment]
Yes. I did sucessfully. There was a running crc in this verysame S.O, but in another Windows account which has been disabled. Im not sure it can be the problem.
I have performet crc delete, crc setup and crc start. I sounds like the deamon is not running.
I tried to start it in another shell before:

PS C:\temp> ./crc daemon
INFO Checking if admin-helper executable is cached
INFO Checking if running on a supported CPU architecture
INFO Checking minimum RAM requirements
INFO Checking if running in a shell with administrator rights
INFO Checking Windows 10 release
INFO Checking Windows edition
INFO Checking if Hyper-V is installed and operational
INFO Checking if user is a member of the Hyper-V Administrators group
INFO Checking if Hyper-V service is enabled
INFO Checking if the Hyper-V virtual switch exist
INFO Found Virtual Switch to use: Default Switch
INFO Checking if vsock is correctly configured
exit status 1
